driverService/src/test/java/com/modsen/driverservice/e2e/steps/DriverStepsDefinitions.java

```diff
@@ -0,0 +1,156 @@
+import com.fasterxml.jackson.databind.ObjectMapper;
+import com.modsen.driverservice.dto.CarRequestDto;
+import com.modsen.driverservice.dto.CarResponseDto;
+import com.modsen.driverservice.dto.DriverRequestDto;
+import com.modsen.driverservice.dto.DriverResponseDto;
+import com.modsen.driverservice.dto.PageDto;
+import io.cucumber.java.en.And;
+import io.cucumber.java.en.Given;
+import io.cucumber.java.en.Then;
+import io.cucumber.java.en.When;
+import io.restassured.response.Response;
+
+import static com.modsen.driverservice.util.E2ETestData.URL_CAR;
+import static com.modsen.driverservice.util.E2ETestData.URL_CAR_DRIVER_ID;
+import static com.modsen.driverservice.util.E2ETestData.URL_CAR_ID;
+import static com.modsen.driverservice.util.E2ETestData.URL_DRIVER;
+import static com.modsen.driverservice.util.E2ETestData.URL_DRIVER_ID;
+import static io.restassured.RestAssured.given;
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+public class DriverStepsDefinitions {
+    private final ObjectMapper objectMapper = new ObjectMapper();
+
+    private Response response;
+    private DriverRequestDto driverRequestDto;
+    private CarRequestDto carRequestDto;
+
+    @Given("Car request dto")
+    public void carRequestDto(String requestBody) throws JsonProcessingException {
+        carRequestDto = objectMapper.readValue(requestBody, CarRequestDto.class);
+    }
+
+    @When("Get page of cars")
+    public void getPageOfCars() {
+        response = given()
+                .when()
+                .get(URL_CAR);
+    }
+
+    @When("Get page of cars by driver id {long}")
+    public void getPageOfCarsByDriverId(long id) {
+        response = given()
+                .when()
+                .get(URL_CAR_DRIVER_ID, id);
+    }
+
+    @When("Get car by id {long}")
+    public void getCarById(long id) {
+        response = given()
+                .when()
+                .get(URL_CAR_ID, id);
+    }
+
+    @When("Create car")
+    public void createCar() {
+        response = given()
+                .contentType("application/json")
+                .body(carRequestDto)
+                .when()
+                .post(URL_CAR);
+    }
+
+    @When("Update car with id {long}")
+    public void updateCarWithId(long id) {
+        response = given()
+                .contentType("application/json")
+                .body(carRequestDto)
+                .when()
+                .put(URL_CAR_ID, id);
+    }
+
+    @When("Delete car with id {long}")
+    public void deleteCarWithId(long id) {
+        response = given()
+                .when()
+                .delete(URL_CAR_ID, id);
+    }
+
+    @And("Response body contains Car response dto")
+    public void responseBodyContainsCarResponseDto(String responseBody) throws JsonProcessingException {
+        CarResponseDto expected = objectMapper.readValue(responseBody, CarResponseDto.class);
+        CarResponseDto actual = response.body().as(CarResponseDto.class);
```

Comment:
Здесь и далее:
```suggestion
        CarResponseDto actual = response.body()
                .as(CarResponseDto.class);
```
Вызовы в цепочке методов следует располагать на отдельных строках.